src/MorseL.Scaleout.Redis/RedisBackplane.cs

```diff
@@ -0,0 +1,237 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Options;
+using MorseL.Common;
+using Newtonsoft.Json;
+using StackExchange.Redis;
+
+[assembly: InternalsVisibleTo("MorseL.Scaleout.Redis.Tests")]
+namespace MorseL.Scaleout.Redis
+{
+    /// <summary>
+    /// Redis backed backplane.
+    /// 
+    /// Group subscriptions work by treating the RedisBackplane as the subscriber for groups.
+    /// The RedisBackplane keeps its own list of groups and subscribers so when a message
+    /// comes in for a group to the RedisBackplane, the RedisBackplane can handle sending
+    /// out the messages itself to the internal clients. Redis allows for multiple handlers
+    /// per subscription but either way some state will have to be tracked.
+    /// 
+    /// It might turn out to be more useful to leverage Redis' implementation but this works
+    /// for now.
+    /// </summary>
+    public class RedisBackplane : IBackplane
+    {
+        private const string RedisKeyAllGroup = "redis-backplane:group:all";
+        private const string RedisKeyConnectionPrefix = "redis-backplane:";

```

Comment:
Well, it could or perhaps I should specify "redis-backplane:connection:" which goes along with what I was intending.